### DISCUSSION TITLE
Lottie file not playing in dotlottie-web

### DISCUSSION THREAD

**Discussion by AlanBreck** [LottieFiles/dotlottie-web#196] [General]
I'm not sure if I'm at liberty to share the file, but I have a Lottie animation which will not play when using `dotlottie-web`. However when using `@lottiefiles/lottie-player`, it works perfectly fine.

UPDATE: it also works perfectly fine in [`@dotlottie/player-component`](https://github.com/dotlottie/player-component/tree/master/packages/player-component).

Are there known reasons for why a valid lottie file wouldn't play with `dotlottie-web`?

**Comment by theashraf** [comment]
@AlanBreck, I'll have a look and let you know. You can email me the animation asset at ashraf@lottiefiles.com if you prefer not to share it here

**Reply by AlanBreck** [reply]
Thank you! Confirming that I can send.

**Comment by theashraf** [comment]
@AlanBreck I believe the main issue is the use of expression functions and layer effects that are not currently supported by our Lottie renderer ([ThorVG](https://github.com/thorvg/thorvg)).You can refer to this [ThorVG issue](https://github.com/thorvg/thorvg/issues/1640) for a list of the currently supported Lottie expressions

**Reply by AlanBreck** [reply]
That's really helpful. Thank you, @theashraf!

**Reply by theashraf** [reply]
@AlanBreck mark as answer ?

**Reply by AlanBreck** [reply]
> @AlanBreck mark as answer ? 

I would! ...but I'm not seeing a button for that. I think maybe because I didn't open it as a Q/A discussion?
